docs/REQUIREMENTS.md

```diff
@@ -7,8 +7,197 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
+- **Tool Writer**: person that builds tools to manage images on registries and other locations.
+- **System Runtime Administrator**: person who is responsible for maintaining the container runtime environment on which OCI containers (e.g. in HPC, just the Syadmin, person deploying cluster or individual runtime nodes)
+- **Deployment Engineer**: person who owns managing the application deployment on to a container runtime cluster/environment/host.
+- **Registry Maintainer**: person maintaining the registry (e.g., Docker Hub, Harbor).
+- **OCI Specification Maintainer**: someone from the open containers initiative (aka us).
+- **Security Administrator**: person responsible for securing the environment.
+
+### User Stories
+
+The following are categorized user stories by personas and roles:
+
+#### Image Author
+
+1. As an image author, I want to update compatibility independently without having to re-release and re-distribute my image.
+2. As an image author, I want to have the freedom to express any compatibility that is necessary for my container to run on the host.
+   - Including conditional compatibility, such as a container running on AMD CPU and intel CPU having different requirements.
+   - Including must have/nice to have compatibility.
+   - Matching finer grain platform definitions.
+3. As an image author, I want to use a provided tool to verify the compatibility spec I wrote against the schema.
+4. As an image author, I would like to create a single compatibility description that is common to a group of images.
+5. As an image author, I want to be able to specify in my manifest that my multi-platform image has a compatibility spec which should be consulted.
+6. As an image author, I want to be able to include compatibility specifications from base layers that I inherited from.
+7. As an image author, I want to ensure that runtimes without image compatibility gracefully fallback to running a usable image.
+
+#### Domain Architect
+
+1. As a domain architect I want a process to share my knowledge about \<niche topic\> compatibility with some compatibility interest group.
+2. As a domain architect I don't want to have to understand containers or develop tools for them to share this knowledge.
+
+#### Tool Writer
+
+1. As a tool writer, I want to get the compatibility spec without pulling the image layer blobs.
+2. As a tool writer, I would like to have a library for reading image compatibility so that I can write my own software that takes action based on the spec, e.g.:
+   - custom k8s scheduler, admission webhooks, runtime classes etc.
+3. As a tool writer, compatibility validation could be integrated into non-runtime tools.
+4. As a tool writer, I want to be able to write tools (that use compatibility specs) that have non-standard applications (e.g., checking individual layers).
+
+#### System Runtime Administrator
+
```

Comment:
What is the difference between "runtime system administrator" and just "system administrator"? Also in the title we say "System Runtime Administrator" but in the bullets we say "runtime system administrator."

Reply:
> What is the difference between "runtime system administrator" and just "system administrator"? 

In the [doc](https://docs.google.com/document/d/1ULXHY0pdiLBlGZPN2Gj54XSvAo--UXKYr4opTrWSFMs/edit) we defined that as:

"System Runtime Administrator: Person who is responsible for maintaining the container runtime environment on which OCI containers (e.g. in HPC, just the Syadmin, person deploying cluster or individual runtime nodes)"

> Also in the title we say "System Runtime Administrator" but in the bullets we say "runtime system administrator."

My bad. I will have to update it.

Reply:
Fixed.